types/web_config.pp

```diff
@@ -0,0 +1,7 @@
+# @summary webconfig for the exporter
+# @see https://github.com/prometheus/exporter-toolkit/blob/v0.10.0/docs/web-configuration.md
+type Prometheus::Web_config = Struct[{
```

Comment:
can you please add unit tests for the datatypes?

Reply:
Added some positive spec tests but no negative ones the test matrix would be huge. Hope that is fine.